sdk/core/System.Net.ClientModel/tests/internal/ModelReaderWriter/ModelJsonConverterTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+using System.Net.ClientModel.Core;
+using System.Net.ClientModel.Tests.Client.ModelReaderWriterTests.Models;
+using System.Text.Json;
+
+namespace System.Net.ClientModel.Tests.Internal.ModelReaderWriterTests
+{
+    public class ModelJsonConverterTests
+    {
+        private readonly JsonSerializerOptions _options;
+
+        public ModelJsonConverterTests()
+        {
+            _options = new JsonSerializerOptions();
+            _options.Converters.Add(new ModelJsonConverter());
```

Comment:
Also non-blocking given timelines, but curious - if ModelJsonConverter is in the public API, would it be possible to move it into the System.Net.ClientModel.Tests project instead of internal?  Before we merge this PR to main, I would like to be a little more rigorous about only having tests in System.Net.ClientModel.Tests.Internal that test internal APIs and have tests for public APIs in the main project.  

If this is properly a test of an internal API, feel free to let me know that that is the case and that I'm just not seeing it yet.